packages/errors/.mocharc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  spec: 'test/unit/*.ts',
```

Comment:
The extension says `js` and `ts` but this is only listing `ts` files. Is that correct?

Reply:
There aren't any `js` files and I don't suspect we would add any, so I removed the `extension`. I'm not even sure if both of these configurations are necessary. Seems duplicative, but I'll just leave it. [`bbfc675` (#31518)](https://github.com/cypress-io/cypress/pull/31518/commits/bbfc67555ed75f12d969840e281678e893cf279d)